packages/shared/src/components/buttons/Button.tsx

```diff
@@ -96,8 +96,8 @@ function ButtonComponent<TagName extends AllowedTags>(
       aria-pressed={pressed}
       ref={ref}
       className={classNames(
-        `btn focus-outline inline-flex cursor-pointer select-none flex-row
-        items-center border no-underline shadow-none transition
+        `btn focus-outline inline-flex cursor-pointer select-none flex-row items-center
+        gap-2 border no-underline shadow-none transition
```

Comment:
I made this change, and the one in buttons/common.ts cause I noticed the icons were always overflowing due to the negative margin:
![Negative margin causes icon to overflow](https://github.com/user-attachments/assets/8737718f-7642-4b5b-b06d-c9d9310bd333)

Reply:
Doesn't affect the other usecases as this is a widely used component?
Maybe you can just add this to the emoji use-case otherwise?

Reply:
Kindly verify whether there was no existing button affected. Although I suggest we not make the change here so we can further test.